data-es5.js

```diff
@@ -1626,10 +1621,33 @@ exports.tests = [
     res: temp.strict,
   },
   {
-    name: 'arguments.caller and arguments.callee is a TypeError',
+    name: 'arguments.caller removed or is a TypeError',
+    exec: function() {/*
+      'use strict';
+      if (Object.getOwnPropertyDescriptor(arguments,'caller')) {
```

Comment:
`getOwnPropertyDescriptor` doesn't exist in ES3 - i think you need to use `in` here.